### ISSUE TITLE
Responsive behavior for TabView

### ISSUE THREAD

**Issue by sousajunior** [primefaces/primeng#5550]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[x] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```
**Current behavior**
Today when you are using the TabView component on a mobile screen the tabs stay one over another. I think that behavior harms the design of the page, because can desalign all page.

**What is the motivation / use case for changing the behavior?**
I had a idea when a saw the google search tabs. They are horizontally scrollable, without desalign the page design.
Its possible to implement the same behavior on TabView when you using on a mobile platforms ?

- Current TabView

![tab1](https://user-images.githubusercontent.com/17458197/38699991-0ae94f92-3e70-11e8-9aa8-fed92dd76869.PNG)

- Example of google tabs

![tab2](https://user-images.githubusercontent.com/17458197/38699992-0b113a52-3e70-11e8-919b-5e2192d7c9f1.PNG)

**Comment by agussantiago** [comment]
For future readers, I implemented sousajunior's comment like this:

```
.responsive-tab-view .p-tabview-nav {
  flex-wrap: nowrap;
  overflow-x: auto;

  li {
    flex: 0 0 auto;
  }
}
```

then when instantiating the tabview:

```
<p-tabView class="responsive-tab-view">
```

**Comment by agustinsantiago** [comment]
For future readers, I implemented sousajunior's comment like this:

```
.responsive-tab-view .p-tabview-nav {
  flex-wrap: nowrap;
  overflow-x: auto;

  li {
    flex: 0 0 auto;
  }
}
```

then when instantiating the tabview:

```
<p-tabView class="responsive-tab-view">
```

**Comment by acsiseci** [comment]
A simple solution method

```
@media (max-width: 991px){
  ul.p-tabview-nav li{
    width: 100% !important;
  }

  .p-tabview .p-tabview-nav li .p-tabview-nav-link {
    border-color: #efefef !important;
    border-top-right-radius: 0px !important;
    border-top-left-radius: 0px !important;
  }

  .p-tabview .p-tabview-nav li.p-highlight .p-tabview-nav-link {
    background: #efefef !important;
    font-weight: bold;
  }
}
```


**Web**
<img width="1584" alt="web" src="https://user-images.githubusercontent.com/10503903/107202505-53ea9880-6a0b-11eb-8519-43fdd0be73f1.png">
**Mobile**
<img width="250" alt="mobile" src="https://user-images.githubusercontent.com/10503903/107202490-4fbe7b00-6a0b-11eb-8d74-d2a050f47a90.png">

**Comment by davisb10** [comment]
This is being resolved on issue #10119

**Comment by mertsincan** [comment]
Yes, fixed in https://github.com/primefaces/primeng/issues/10119

Best Regards,
